snippet_parser/core_test.py

```diff
@@ -146,5 +146,28 @@ def test_no_wikicode_in_section_titles(self):
             "== Section title ''with Wikicode'' == \n\nIrrelevant content{{cn}}")
         self.assertEqual(section, 'Section title with Wikicode')
 
+    def test_multiple_snippets_in_section(self):
+        _, [section, snippets] = self._do_extract(
+            '<p>This needs a reference{citation_needed_tmpl}</p>'
+            '<p>So does this{citation_needed_tmpl}',
```

Comment:
It seems missing a `</p>` ? it doesn't affect the result though. 

Btw, I just realized in Python you can pass multiple strings to one parameter without concatenating them using `+`.. Python will regard them as one string! :smiley:

Reply:
Nice catch! And yeah, Python [concatenates string literals](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str) if they are only separated by space, anywhere in the program. So here we're really just passing a single string to the method, which is the equivalent of using `+`!